### PR TITLE
Bug 1215095 - calendar css refactoring

### PR DESCRIPTION
**Pull request by peppsac**
[Review on Reviewable](https://reviewable.io/reviews/mozilla-b2g/gaia/32592)

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=229eab3e8000940f140ce310fb3ff73b29c31ec1

File: apps/calendar/style/multi_day.css [hunk 1]
```diff
@@ -393,14 +389,14 @@ html[dir="rtl"] .md__event.is-partial-small .md__event-title + .md__event-locati
...
   position: absolute;
-  top: 0;
+  top: 0.25rem;
   width: 0;
   height: 0;
   border-style: solid;
-  border-width: 0.4rem 0 0.4rem 0.6rem;
-  border-color: transparent transparent transparent #008EAB;
-  /* scale is for smooth edges and translate for centering the line */
-  transform: scale(0.99) translateY(0.25rem);
```
Comment (sfoster) [inline apps/calendar/style/multi_day.css]: why remove this transform? It may or may not be strictly necessary but seems orthogonal to this BiDi refactor?

Comment (peppsac) [inline apps/calendar/style/multi_day.css]: Sorry missed your comment.
I removed it as a cleanup while touching this file: rendering with or without the scale(0.99) was identical.
